app/routes/index.tsx

```diff
@@ -1,11 +1,52 @@
-import { LinksFunction } from '@remix-run/node';
+import { LinksFunction, LoaderFunction } from '@remix-run/node';
 import styles from 'react-datepicker/dist/react-datepicker.css';
 import Calendar from '~/components/Calendar';
 import Navbar from '~/components/common/navbar';
+import axios from 'axios';
+import moment from 'moment';
+import { useStore } from '~/store/useStore';
+import { useLoaderData } from '@remix-run/react';
+import { parseEvents } from '~/utils/event.utils';
+import { getEvents } from '~/constants/urls.constants';
+import { ToastContainer, toast } from 'react-toastify';
+import { useEffect } from 'react';
+
+export let loader: LoaderFunction = async ({ request }) => {
+  const cookie = request.headers.get('cookie');
+
+  const startTime = moment().subtract(1, 'months').startOf('month').unix() * 1000;
+  const endTime = moment().add(1, 'months').endOf('month').unix() * 1000;
+
+  try {
+    const response = await axios.get(getEvents(process.env.API_HOST ?? '', startTime, endTime), {
+      headers: {
+        'Content-Type': 'application/json',
+        Cookie: cookie,
+      },
+    });
+    return { events: response.data.data, error: null };
```

Comment:
Do we always get response.data.data
Is it validated through backend and we have defined interface for the same?

Reply:
we're getting the response from axios in `response.data`, then in response we have it in data as well. Here's a sample response: 

```
{
  data: [
    {
      id: 14,
      name: 'new event',
      description: null,
      location: null,
      startTime: '2023-04-27T22:43:20.000Z',
      endTime: '2023-04-27T23:50:00.000Z',
      ownerId: 4,
      eventTypeId: 1,
      calendarId: 1,
      isDeleted: false,
      Attendees: [Array]
    }
  ]
}
```
that's why `response.data.data`.

PS: We should create a swagger page for it so that we can see it all there.